components/layout_2020/flow/construct.rs

```diff
@@ -639,9 +677,3 @@ impl std::ops::BitOrAssign for ContainsFloats {
         }
     }
 }
-
-impl Default for ContainsFloats {
-    fn default() -> Self {
-        ContainsFloats::No
-    }
-}
```

Comment:
Why remove this impl?

Reply:
It was only used in a code that called `Default::default()` to initialize three struct fields, and I felt that having type-specific constructors like `Vec::new()` and `None` made that code easier to read.